queries/buffered_land.jinja2

```diff
@@ -0,0 +1,28 @@
+SELECT
+    gid AS __id__,
+    ST_AsBinary(geom) AS __geometry__,
+    jsonb_build_object(
+      'source', 'tilezen.org',
+      'min_zoom', 0,
+      'kind', 'maritime',
+      'maritime_boundary', TRUE
+    ) AS __properties__,
+
+    '{}'::jsonb AS __boundaries_properties__
+
+FROM (
+  SELECT
+    gid,
+    -- extract only polygons. we might get linestring and point fragments when
+    -- the box and geometry touch but don't overlap. we don't want these, so
+    -- want to throw them away.
+    fiahdlfauisdhluhST_CollectionExtract({% filter geometry %}the_geom{% endfilter %}, 3) AS geom
```

Comment:
I assume this is a frustration typo? :)

Reply:
D'oh! Thanks for spotting that! Removed in 85a54fb.

It wasn't frustration, more that I was testing the RAWR vs. SQL paths and wanted to make absolutely certain that the data was coming from the RAWR tile. I figured an easy way to check that was to have PostgreSQL tell me that function didn't exist. Of course, then I forgot to take it out. :man_facepalming: